lib/install/realize-shrinkwrap-specifier.js

```diff
@@ -1,14 +1,15 @@
 'use strict'
 var realizePackageSpecifier = require('realize-package-specifier')
 var isRegistrySpecifier = require('./is-registry-specifier.js')
+var url = require('url')
 
 module.exports = function (name, sw, where, cb) {
   function lookup (ver, cb) {
     realizePackageSpecifier(name + '@' + ver, where, cb)
   }
   if (sw.resolved) {
     return lookup(sw.resolved, cb)
-  } else if (sw.from) {
+  } else if (sw.from && url.parse(sw.from).protocol) {
```

Comment:
On second thought, that's basically what the code in here does already. I think we need to dig further into the root cause here, which I believe would be that `isRegistrySpecifier` is doing the wrong thing somehow? Which implies that in turn `realizePackageSpecifier` is doing the wrong thing?